bitnami/argo-workflows/values.yaml

```diff
@@ -0,0 +1,1025 @@
+  ## @param controller.readinessProbe.enabled Enable readinessProbe on controller nodes
+  ## @param controller.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param controller.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param controller.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param controller.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param controller.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 90
+    periodSeconds: 60
+    timeoutSeconds: 30
+    failureThreshold: 3
+    successThreshold: 1
+  ## @param controller.customLivenessProbe Custom livenessProbe that overrides the default one
+  ##
+  customLivenessProbe: {}
+  ## @param controller.customReadinessProbe Custom readinessProbe that overrides the default one
+  ##
+  customReadinessProbe: {}
+  ## controller resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param controller.resources.limits The resources limits for the controller containers
+  ## @param controller.resources.requests The requested resources for the controller containers
+  ##
+  resources:
+    limits: {}
+    requests: {}
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param controller.podSecurityContext.enabled Enabled controller pods' Security Context
+  ## @param controller.podSecurityContext.fsGroup Set controller pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param controller.containerSecurityContext.enabled Enabled controller containers' Security Context
+  ## @param controller.containerSecurityContext.runAsUser Set controller containers' Security Context runAsUser
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+
+  ## Server container port
+  ## @param controller.ports.container Argo Workflows controller container port
+  ports:
+    container: 6060
+
+  ## Controller persistence parameters. It can also be connected to an existing postgresql. Check
+  ## @param controller.persistence.enabled Enabled controller persistence
+  ## @param controller.persistence.connectionPool.maxIdleConns Msximun number of postgresql connections
+  ## @param controller.persistence.connectionPool.maxOpenConns Maximun number of open connections
+  ## @param controller.persistence.nodeStatusOffLoad Save the entire workflow into etcd and DB
+  ## @param controller.persistence.archive Enable archiving old workflows
+  ## @param controller.persistence.postgresql.enabled Enable Postgresql persistence
```

Comment:
I see this is actually part of the configuration file. I don't like mixing parts of the chart values with the application configuration (it is very confusing in terms of maintenance). Plus, we should find a similar user experience when dealing with sub-charts. Feel free to take a look to the JupyterHub case.